crates/sc-proof-of-time/src/source.rs

```diff
@@ -28,12 +28,12 @@ pub struct PotSlotInfo {
 pub struct PotSlotInfoStream(mpsc::Receiver<PotSlotInfo>);
 
 /// Configuration for proof of time source
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug, Clone)]
 pub struct PotSourceConfig {
     /// Is this node a Timekeeper
     pub is_timekeeper: bool,
-    /// PoT key used initially when PoT chain starts
-    pub initial_key: PotKey,
+    /// External entropy, used initially when PoT chain starts to derive the first seed
+    pub external_entropy: Vec<u8>,
```

Comment:
Do we not know the size of the entropy already? I have seen the discussion you started on unifying seed and key but did not follow to the end

Reply:
Entropy will likely end up being Bitcoin block hash, but technically it doesn't have to be, so I decided to use vector of bytes for now. It is used together with genesis block hash to derive initial seed, key is always derived from seed later.